### PR TITLE
[MPDX-8088] Create setup start page

### PR DESCRIPTION
**Pull request by canac**
## Description

* Created a setup start page at `/setup/start` that lets the user pick their language.
* A future PR will handle automatically detecting when the user needs to start the setup tour.
* Added environment variables to Amplify.

https://jira.cru.org/browse/MPDX-8088

## Checklist:

- [x] I have given my PR a title with the format "MPDX-(JIRA#) (summary sentence max 80 chars)"
- [x] I have applied the appropriate labels. (_Add the label "On Staging" to get the branch automatically merged into staging._)
- [x] I have requested a review from another person on the project

### CODE REVIEW

**Review by dr-bizz — CHANGES_REQUESTED** [review]
The code looks great! It still needs work on the design as it looks a little off compared to the rest of the design on MPDX.

Screenshot 2024-08-23 at 10 18 00 AM

File: src/components/Links/Links.test.tsx
[the remark this answers is not in the document]
Comment (canac) [reply src/components/Links/Links.test.tsx:1]: Sure thing, I moved the components into `Shared`.

**Comment by canac** [comment]
> The code looks great! It still needs work on the design as it looks a little off compared to the rest of the design on MPDX.

@dr-bizz I don't disagree that there's room for improvement. Let me know if you have any specific suggestions.

@j2trumpet Do you have any ideas for improving the design? I could put it in a card and/or add a blue background, but I'm at a loss besides those ideas. The Angular version looks like the screenshot in [this ticket](https://jira.cru.org/browse/MPDX-8088).

**Comment by j2trumpet** [comment]
Nothing jumps out at me as looking bad. Looking at the Jira ticket, it looks like this screen is all the content on the page rather than a modal appearing over other content. Are there other screens during the account/app setup process that this language screen could be styled like?

**Comment by canac** [comment]
> Are there other screens during the account/app setup process that this language screen could be styled like?

@j2trumpet The connect organization screen looks similar (https://jira.cru.org/browse/MPDX-8089). I'll definitely make these two screens look similar.

**Comment by canac** [comment]
@dr-bizz I made the setup content a Card on a blue background. Let me know what you think.

**Review by dr-bizz — CHANGES_REQUESTED** [review]
I like the changes you made, it's starting to look great. I still think there is some room for improvement.

What are your thoughts to making it like this:

Screenshot 2024-08-27 at 8 27 57 AM

**Comment by canac** [comment]
@dr-bizz Does your comment on the other PR list the styles you changed to get that screenshot? I see you also made the button full-width. Did you change anything else?

> Header text: font-size: 2.75rem
> Icon colour: primary
> Paragraph - max-width: 500px
> Card max-width: 580px

**Comment by dr-bizz** [comment]
> @dr-bizz Does your comment on the other PR list the styles you changed to get that screenshot? I see you also made the button full-width. Did you change anything else?
> 
> > Header text: font-size: 2.75rem
> > Icon colour: primary
> > Paragraph - max-width: 500px
> > Card max-width: 580px

I also made all the input and paragraphs in line with one another. But I think that is it.

**Review by dr-bizz — APPROVED** [review]
Looks awesome! Great job

**Comment by canac** [comment]
> Oh just thought on the start pages, is it worth changing the colout of the help icon to something other than blue so it's more visible? - just a thought

Good idea. We're switching to HelpJuice pretty soon (I was working on the PR yesterday), and the HelpJuice beacon color is more different from the MPDX background color, so I'll probably leave it as-is if that's OK. And if the contrast doesn't end up being good enough, I can after that.

Screenshot 2024-08-28 at 11 02 45 AM

**Pull request merged**